src/toil/batchSystems/lsfHelper.py

```diff
@@ -21,22 +21,23 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
 import fnmatch
-import math
 import os
 import re
 import subprocess
 
 from packaging import version
 
+from toil.wdl.wdl_functions import return_bytes
```

Comment:
Yay for code reuse! Shouldn't `return_bytes` now be placed somewhere more generic than the WDL package?

Reply:
Good point!  I've generalized it now.